x-pack/server/lib/check_license/__tests__/check_license.js

```diff
@@ -10,46 +10,40 @@ import { checkLicense } from '../check_license';
 
 describe('check_license', function () {
 
+  const pluginName = 'Foo';
+  const minimumLicenseRequired = 'basic';
   let mockLicenseInfo;
   beforeEach(() => mockLicenseInfo = {});
 
   describe('license information is undefined', () => {
     beforeEach(() => mockLicenseInfo = undefined);
 
     it('should set isAvailable to false', () => {
-      expect(checkLicense(mockLicenseInfo).isAvailable).to.be(false);
+      expect(checkLicense(pluginName, minimumLicenseRequired, mockLicenseInfo).isAvailable).to.be(false);
     });
 
-    it('should set showLinks to true', () => {
-      expect(checkLicense(mockLicenseInfo).showLinks).to.be(true);
-    });
-
-    it('should set enableLinks to false', () => {
-      expect(checkLicense(mockLicenseInfo).enableLinks).to.be(false);
+    it('should set status to unavailable', () => {
+      expect(checkLicense(pluginName, minimumLicenseRequired, mockLicenseInfo).status).to.be('UNAVAILABLE');
```

Comment:
Since we have the constants file, can we consume that instead of hardcoding these strings?